src/utils/helpers.js

```diff
@@ -70,6 +70,16 @@ export const getFilenameFromPath = (path) => {
 };
 
 /**
+ * Returns extension from the given path or filename
+ * @param {String} path or filename
+ * @return {String} extension
+ */
+export const getExtensionFromPath = (path) => {
+  if (!path) return '';
+  return path.substring(path.lastIndexOf('.') + 1);
+};
+
```

Comment:
added in [`1fcc613`](https://github.com/jekyll/jekyll-admin/pull/360/commits/1fcc613859e42a8cb47ba15de865ec65fb3091d2) .. changed the function a bit to account for paths that do not contain an extension